episodes/imaging-software.md

```diff
@@ -315,8 +317,8 @@ various aspects of the Napari viewer:
 https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/console.svg
 ){alt="A screenshot of Napari's console button" height='30px'}
 
-This button opens Napari's built-in python console - we'll look at this in later 
-episodes.
+This button opens Napari's built-in python console - we'll look at this in the 
```

Comment:
This link doesn't really describe what the python console is, rather it links to an example of us using the python console. If the user follows this link they might be disappointed. Maybe it would just be better to say 'We'll use the console more in later episodes' and not bother with a link.

Reply:
Thanks @thompson318 - I removed the link for this line. 

If you're happy with this, could you approve + merge?